### DISCUSSION TITLE
Failed to Execute Powershell Script: AuthorizationManager check failed 🤔

### DISCUSSION THREAD

**Discussion by FynnNikolaus** [renovatebot/renovate#14395] [Request Help]
Hi guys, unfortunately after updating from **Cake version 1.1.0 to Cake 2.0.0** I received the following error message after running a task that starts a powershell script:
![CakeError](https://user-images.githubusercontent.com/48774073/155526264-94353b61-f4ae-4cfd-9241-05be8ddd9e02.jpg)

All other tasks execute, only the task in which a powershell script is invoked now fails with the new version.
![CakeTask](https://user-images.githubusercontent.com/48774073/155527138-7a4c949f-8217-49d9-a868-fa46b8db5b8f.jpg)

Many thanks for your help and suggestions 🐱‍💻

**Comment by viceice** [comment]
I think you should check the changes for cake v2.0. it seems they have breaking changes. Those are out of scope for renovate. So please ask at cake community for required changes.
